packages/components/psammead-most-read/src/Item/__snapshots__/index.test.jsx.snap

```diff
@@ -311,6 +311,7 @@ exports[`MostReadItemWrapper should render ltr correctly with 10 items 1`] = `
 
 <div>
   <li
+    aria-labelledby="1 Could a computer ever create better art than a human?"
```

Comment:
Hey! I'm not sure this is the correct usage of the `aria-labelledby` tag - that tag should be used to associate an element with another element that acts as its label, using the ID of the labelling element. What problem are you solving here by using this tag?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute